test/passes/trace-calls.wast

```diff
@@ -0,0 +1,45 @@
+(module
+  (import "env" "no_params_no_results" (func $noparamsnoresults))
+
+  (import "env" "single_param_no_results"
+    (func $singleparamnoresults (param f64)))
+  (import "env" "multi_params_no_results"
+    (func $multiparamsnoresults (param i32 i64 f32)))
+
+  (import "env" "no_params_single_result"
+    (func $noparamssingleresult (result v128)))
+
+  (import "env" "multi_params_single_result"
+    (func $multiparamssingleresult (param i32 v128)(result v128)))
+
+  (import "env" "dont_trace_me" (func $donttraceme))
+
+  (func $test_no_params_no_results
+    (call $noparamsnoresults)
+  )
+
+  (func $test_single_param_no_results
+    (call $singleparamnoresults (f64.const 4.5))
+  )
+  ;; we specify a custom name (tracempnr) for the tracer function
+  (func $test_multi_params_no_results
+    (call $multiparamsnoresults
+      (i32.const 5)
+      (i64.const 6)
+      (f32.const 1.5)))
+
+  (func $test_no_params_single_result (result v128)
+    call $noparamssingleresult
+  )
+
+  (func $test_multi_params_single_result (result v128)
+    (call $multiparamssingleresult
+      (i32.const 3)
+      (v128.const i32x4 1 2 3 4))
+  )
+
+  ;; this function should not be traced
+  (func $test_dont_trace_me
+    call $donttraceme
+  )
+)
```

Comment:
We are moving tests from `passes/` to `lit/passes/`, and converting them to lit, so new passes should go there, sorry for the confusion.

If you're not familiar with LLVM lit then copy-pasting from other files in that directory should be easy enough.

Reply:
no problem, done.